backend/typescript/services/implementations/emailService.ts

```diff
@@ -34,7 +34,8 @@ class EmailService implements IEmailService {
     try {
       return await this.transporter.sendMail(mailOptions);
     } catch (error) {
```

Comment:
```suggestion
    } catch (error: unknown) {
```

Might be better to just explicitly type it so that unless the if statement is given, vscode will complain

Reply:
ok I'll give it a try

Reply:
Actually keep this unchanged, i change this file as a part of this PR: https://github.com/uwblueprint/ccbc/pull/17